Doc/library/tarfile.rst

```diff
@@ -1346,6 +1346,15 @@ parameter in :meth:`TarFile.add`::
     tar.add("foo", filter=reset)
     tar.close()
 
+How to create and write an archive to stdout using
+:data:`sys.stdout.buffer <sys.stdout>` in the *fileobj* parameter
+in :meth:`TarFile.add`::
+
+    import sys
+    import tarfile
+    with tarfile.open("sample.tar.gz", "w:gz", fileobj=sys.stdout.buffer) as tar:
```

Comment:
Does it matter if we use `w:gz` or `w|gz` here?

Reply:
Streaming mode is probably better to recommend, yes.

A